setup.cfg

```diff
@@ -56,6 +55,7 @@ rst-directives =
     versionchanged,
     todo,
 rst-roles =
+    attr,
```

Comment:
I had to add this here or I got errors about an unknown role. `:attr:` should be used to reference attributes and properties.